eng/testing/linker/project.csproj.template

```diff
@@ -56,4 +56,5 @@
                       IsImplicitlyDefined="true" />
   </ItemGroup>
 
+  <Import Project="{NativeSanitizersTargets}" />
```

Comment:
Why is this needed in the linker case?

Reply:
We reuse the "linker" test infrastructure for NativeAOT and we need it for NativeAOT.